<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTPSettings</class>
 <widget class="QWidget" name="FTPSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>380</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>FTP设置</string>
  </property>
  <widget class="QLineEdit" name="DEVICE_IP_EDT">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="FTP_PWD_EDT">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="FTP_SERVER_EDT">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="FTP_USR_EDT">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DEVICE_IP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>52</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设备地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="FTP_SERVER">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FTP服务器</string>
   </property>
  </widget>
  <widget class="QLabel" name="FTP_USR">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FTP用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="FTP_PWD">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>49</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FTP密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NANOPI_IP_EDT">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>20</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="GET_CONFIG_BTN">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>读取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RESET_BTN">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>恢复默认</string>
   </property>
  </widget>
  <widget class="QLabel" name="NANOPI_IP">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>65</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开发板地址</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="SET_CONFIG_FTP_BTN">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置FTP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CUSTOM_CMD_EDT">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CONNECT_TO_NANOPI_BTN">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SEND_CMD_BTN">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="CUSTOM_CMD">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>26</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>命令</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="GATEWAY">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>26</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>网关</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="STATIC_IP_EDT">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SUBNET_MASK">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>52</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>子网掩码</string>
   </property>
  </widget>
  <widget class="QLabel" name="STATIC_IP">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>26</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SUBNET_MASK_EDT">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="GATEWAY_EDT">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DNS_SERVER">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>28</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DNS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DNS_SERVER_EDT">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="SET_CONFIG_IP_BTN">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UPLOAD_INTERVAL_EDT">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="UPLOAD_INTERVAL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>52</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上传间隔</string>
   </property>
  </widget>
  <widget class="QLabel" name="UPLOAD_INTERVA_UNIT">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>26</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>分钟</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="WORKING_START_EDT">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="WORKING_END_EDT">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="WORKING_HOURS">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>52</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上传时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>320</y>
     <width>6</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
